spec/services/distribution_update_service_spec.rb

```diff
@@ -1,14 +1,48 @@
 RSpec.describe DistributionUpdateService, type: :service do
-  describe "call" do
-    # TODO: this function was extracted to the service object - do we need a parallel test?
-
-    let!(:distribution) { FactoryBot.create(:distribution, :with_items, item_quantity: 10) }
-    let!(:new_attributes) { { line_items_attributes: { "0": { item_id: distribution.line_items.first.item_id, quantity: 2 } } } }
+  subject { DistributionUpdateService }
+  let!(:storage_location) { create(:storage_location, :with_items) }
+  let!(:distribution) { FactoryBot.create(:distribution, :with_items, item_quantity: 10) }
+  let!(:new_attributes) { { line_items_attributes: { "0": { item_id: distribution.line_items.first.item_id, quantity: 2 } } } }
```

Comment:
I think we can change these to `let` instead of `let!` since the variables get defined when you do `subject.new(...)`.

Reply:
I'll give that a shot :+1: